utils/autotailor

```diff
@@ -339,7 +367,9 @@ def get_parser():
     parser.add_argument(
         "-j", "--json-tailoring", metavar="JSON_TAILORING_FILENAME", default="",
         help="JSON Tailoring (https://github.com/ComplianceAsCode/schemas/blob/main/tailoring/schema.json) "
-        "filename.")
+        "filename. When a JSON tailoring is accompanied with additional adjustments from the command-line"
```

Comment:
I suggest improving the help text to clarify that the JSON is an input file that needs to be created and supplied by the user ie. it isn't an output file generated by autotailor.

Reply:
Text has been improved.